tests/acceptance/features/apiSpacesDavOperation/copyByFileId.feature

```diff
@@ -148,3 +148,39 @@ Feature: copying file using file id
       | dav-path                          |
       | /remote.php/dav/spaces/<<FILEID>> |
       | /dav/spaces/<<FILEID>>            |
+
+
+  Scenario Outline: copy a file from personal to project space
+    Given using spaces DAV path
```

Comment:
```suggestion
```
Already in background. Also remove from others as well

Reply:
okay